client/my-sites/customer-home/main.jsx

```diff
@@ -44,6 +46,11 @@ const Home = ( {
 
 	const { data: layout, isLoading } = useHomeLayoutQuery( siteId );
 
+	const { isModalOpen, openModal, closeModal } = useRouteModal(
```

Comment:
Can we reuse these for the modal displayed on the earn card? Otherwise, I feel like we're duplicating a bunch of code.

Reply:
Ok, removed VideosUI from My Home Earn Card.